tools/topology/sof-cml-eq-rt1011-rt5682.m4

```diff
@@ -0,0 +1,70 @@
+#
+# Topology for Cometlake with rt1011 spk on SSP1
+#
+
+# Include SOF CML RT5682 Topology
+# This includes topology for RT5682, DMIC and 3 HDMI Pass through pipeline
+include(`sof-cml-eq-rt5682-kwd.m4')
+include(`abi.h')
+DEBUG_START
+#
+# Define the Speaker pipeline
+#
+# PCM5  ----> volume (pipe 7) ----> SSP1 (speaker - rt1011, BE link 5)
+#
+
+dnl PIPELINE_PCM_ADD(pipeline,
+dnl     pipe id, pcm, max channels, format,
+dnl     period, priority, core,
+dnl     pcm_min_rate, pcm_max_rate, pipeline_rate,
+dnl     time_domain, sched_comp)
+
+# Low Latency playback pipeline 7 on PCM 5 using max 2 channels of s32le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+	7, 5, 2, s32le,
+	1000, 0, 0,
+	48000, 48000, 48000)
+
+#
+# Speaker DAIs configuration
+#
+
+dnl DAI_ADD(pipeline,
+dnl     pipe id, dai type, dai_index, dai_be,
+dnl     buffer, periods, format,
+dnl     deadline, priority, core, time_domain)
+
+# playback DAI is SSP1 using 2 periods
+# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+DAI_ADD(sof/pipe-dai-playback.m4,
+	7, SSP, 1, SSP1-Codec,
+	PIPELINE_SOURCE_7, 2, s24le,
+	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+
+# PCM Low Latency, id 0
+dnl PCM_PLAYBACK_ADD(name, pcm_id, playback)
+PCM_PLAYBACK_ADD(Speakers, 5, PIPELINE_PCM_7)
+
+#
+# BE configurations for Speakers - overrides config in ACPI if present
+#
+
+#SSP 1 (ID: 6)
+#Use BCLK delay in SSP_CONFIG_DATA only on supporting version
+
+ifelse(SOF_ABI_VERSION_3_9_OR_GRT, `1',
+`DAI_CONFIG(SSP, 1, 6, SSP1-Codec,
```

Comment:
Conditional logic again. The only difference between the two configs is SSP_CONFIG_DATA, can we define SSP_CONFIG_DATA in PLATFORM.m4 (which will have the ABI check)

Reply:
@lgirdwood thank you for pin-pointing the problem. I will fix and resubmit the PR.

Reply:
Sorry for delay. Still working on a patch. Struggling to fix some differences after my new patch. Trying to finalizing the patch soon.

Reply:
I submitted https://github.com/thesofproject/sof/pull/2530. I was debugging some issues, I decided better share my finds and get input from reviewers. If #2530 is revised and approved, I plan to update this also. Now I can see much bigger picture, there is no point to create new file for one line change. I will add makefile variable and put some conditions.